Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeViewController.swift

```diff
@@ -695,14 +693,16 @@ class CreateSafeViewController: UIViewController, UITableViewDelegate, UITableVi
     @IBAction func didTapPaymentMethod(_ sender: Any) {
         // open payment method selection
         let choosePaymentVC = ChoosePaymentViewController()
-        choosePaymentVC.relaysRemaining = relaysRemaining
-        choosePaymentVC.relaysLimit = relaysLimit
+        choosePaymentVC.relaysRemaining = uiModel.relaysRemaining
+        choosePaymentVC.relaysLimit = uiModel.relaysLimit
         choosePaymentVC.userSelectedSigner = uiModel.userSelectedPaymentMethod == .signerAccount
 
         choosePaymentVC.chooseRelay = { [unowned self] in
             LogService.shared.debug("User selected Relay")
             executionOptionsCellBuilder.userSelectedSigner = false
-            uiModel.userSelectedPaymentMethod = .relayer
+            if chain.isSupported(feature: .relayingMobile) && uiModel.relaysRemaining > ReviewExecutionViewController.MIN_RELAY_TXS_LEFT) {
```

Comment:
Extra safeguard which is probably not necessary as this option should not be enabled in the UI already.